ext/session/session.c

```diff
@@ -2381,6 +2380,31 @@ static PHP_FUNCTION(session_unset)
 }
 /* }}} */
 
+/* {{{ proto int session_gc(void)
+   Perform GC and return number of deleted sessions */
+static PHP_FUNCTION(session_gc)
+{
+	zend_long nrdels;
```

Comment:
Fine with me. I'm using nrdels because the name is used for "number of deleted sessions" in the code.